stable/insights-agent/templates/kube-hunter/rbac.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.kubehunter.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-kube-hunter
+  labels:
+    app: insights-agent
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
I think we can delete this ClusterRole and ClusterRoleBinding - kube-hunter doesn't seem to need any RBAC. Let's just use an empty service account